### PR TITLE
Added apis

### PR DESCRIPTION
**Pull request by jamidt**

### CODE REVIEW

**Comment by jamidt** [comment]
Looks fine.

**Comment by Echsecutor** [comment]
I have now briefly read the Code, looks very good, thanks!

 [ ] would it be possible to write a couple of tests for those APIs not requiring keys?
 [ ]  maybe BCdeSession should als derive from BaseQuerry?
 [ ] do the ShapeshiftTransactions not include any rate? Getting only the amount might not be to interesting. ;)
 [ ] the new models are not yet used, are they? E.g. `ShapeshiftMarketinfo` is not instantiated.

**Comment by Echsecutor** [comment]
this MR adresses the 2. part of https://github.com/Echsecutor/bitcoin_trading.echse/issues/6

**Review by Echsecutor — CHANGES_REQUESTED** [review]
I have now briefly read the Code, looks very good, thanks!

- would it be possible to write a couple of tests for those APIs not requiring keys?
- maybe BCdeSession should als derive from BaseQuerry?
- do the ShapeshiftTransactions not include any rate? Getting only the amount might not be to interesting. ;)
- the new models are not yet used, are they? E.g. ShapeshiftMarketinfo is not instantiated.

**Review by Echsecutor — COMMENTED** [review]
> - maybe BCdeSession should als derive from BaseQuerry?
:white_check_mark:

File: webgui/price_chart/apis.py [hunk 1]
```diff
@@ -30,7 +30,30 @@
...
+        if headers:
+            logging.debug("headers=%s", headers)
+        response = requests.get(url, headers=headers)
+        if response.status_code != 200:
+            logging.error("Error getting info. Response %s", response.status_code)
+            return None
+        else:
+            return response
+
+    def query_json(self, url, *args, **kwargs):
```
Comment (Echsecutor) [inline webgui/price_chart/apis.py]: @jamidt  why the `*args, **kwargs` arguments? (unused warning seems ustified)

**Review by Echsecutor — APPROVED** [review]
@jan: Cool, thanks!
I started writing the ETL and adapting the viewers

**Comment by Echsecutor** [comment]
:white_check_mark: > do the ShapeshiftTransactions not include any rate? Getting only the amount might not be to interesting. 

:white_check_mark: > the new models are not yet used, are they? E.g. ShapeshiftMarketinfo is not instantiated.

**Pull request merged**